src/System.Web.OData/Settings.StyleCop

```diff
@@ -1,29 +1,15 @@
 <StyleCopSettings Version="105">
+  <GlobalSettings>
```

Comment:
Why are you modifying the stylecop configuration?

Reply:
Sorry my mistake.

Reply:
This file is still included in the commit, did you plan to revert it?